SRC/STUDIO/ESL.CPP

```diff
@@ -1054,7 +1054,7 @@ bool ESLA::_FAcceptChanges(bool *pfDismissEasel)
 		{
 		if (_pape->FGetMaterial(ibset, &fMtrl, &cmid, &tagMtrl))
 			{
-			if (!_pactr->FSetCostumeCore(ibset, &tagMtrl, cmid, !fMtrl))
+			if (!_pactr->FSetCostumeCore(ibset, &tagMtrl, cmid, (tribool)!fMtrl))
```

Comment:
inferno take here but maybe we should add an `operator !` overload for tribool... 😅